cmd/resourcesnapshot/resourcesnapshot_update.go

```diff
@@ -0,0 +1,91 @@
+package resourcesnapshot
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/civo/civogo"
+	"github.com/civo/cli/common"
+	"github.com/civo/cli/config"
+	"github.com/civo/cli/utility"
+	"github.com/spf13/cobra"
+)
+
+var name, description string
+
+var resourceSnapshotUpdateCmd = &cobra.Command{
+	Use:     "update ID/NAME",
+	Args:    cobra.MinimumNArgs(1),
+	Aliases: []string{"modify", "change"},
+	Short:   "Update resource snapshot details",
+	Long: `Update the name or description of a resource snapshot by ID or name.
+If you wish to use a custom format, the available fields are:
+
+* id
+* name
+* description`,
+	Run: func(cmd *cobra.Command, args []string) {
+		utility.EnsureCurrentRegion()
+
+		client, err := config.CivoAPIClient()
+		if common.RegionSet != "" {
+			client.Region = common.RegionSet
+		}
```

Comment:
all these common.RegionSet could be in the PreRun of the cobra.Command maybe?
also i think there could be some better way to not write this check every time, it's a bit error prone since people could forget to write it. Not sure, what do you think?

Reply:
very true, in my last PR i forgot adding it which flagged up bugs. I'll find some way to refactor this in another PR since this will be a codebase wide change